content/numpy/concepts/built-in-functions/terms/trace/trace.md

```diff
@@ -0,0 +1,144 @@
+---
+Title: '.trace()'
```

Comment:
Instead of  ".trace()" we can use "numpy.trace()" to indicate the exact many of NumPy's trace functions used.